src/Microsoft.AspNetCore.Build/scripts/KoreBuild.ps1

```diff
@@ -150,6 +150,33 @@ if(Test-Path $MSBuildLog) {
     del $MSBuildLog
 }
 
+$MSBuildArguments = @"
+/nologo
+"$Proj"
+/p:KoreBuildToolsPackages="$ToolsDir"
+/p:KoreBuildTargetsPath="$KoreBuildTargetsRoot"
+/p:KoreBuildTasksPath="$MSBuildDir\bin\pub"
+/fl
+/flp:LogFile="$MSBuildLog";Verbosity=diagnostic;Encoding=UTF-8
```

Comment:
Verbosity should be configurable and I don't think it should be diagnostic by default